### PR TITLE
[netatmo] Correcting boiler_connected swagger reading

### PR DESCRIPTION
**Pull request by clinique**

### CODE REVIEW

**Comment by clinique** [comment]
Previous correction has been merged a bit too quick (...while having being opened for a while :) 
There was a problem in the swagger description of boiler_connected

File: addons/binding/org.openhab.binding.netatmo/build.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@ bin.includes = META-INF/,\
                lib/okio-1.3.0.jar,\
                lib/json-20131018.jar,\
                lib/commons-codec-1.8.jar,\
-               lib/gson-2.3.1.jar,\
                about.html,\
-               lib/netatmo-java-retrofit-1.0.8.jar
+               lib/netatmo-java-retrofit-1.0.9.jar,\
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.netatmo/build.properties]: Git suggests (for the binary) that its renamed, which suggests that its same file, can you check it out?

**Comment by martinvw** [comment]
> Previous correction has been merged a bit too quick (...while having being opened for a while :)
There was a problem in the swagger description of boiler_connected

Okay I had people complaining why it was not merged 😜 

Can someone besides @clinique please test and confirm!

**Comment by lolodomo** [comment]
@clinique : please adjust the description of updated channels in the French translation.

File: addons/binding/org.openhab.binding.netatmo/build.properties [hunk 2]
```diff
@@ -13,5 +13,5 @@ bin.includes = META-INF/,\
                lib/commons-codec-1.8.jar,\
                lib/gson-2.3.1.jar,\
                about.html,\
-               lib/netatmo-java-retrofit-1.0.8.jar
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.netatmo/build.properties]: Did you remove the actual file?

**Comment by martinvw** [comment]
@clinique can you ping me if the solution is tested and confirmed by the user?

File: addons/binding/org.openhab.binding.netatmo/build.properties (hunk 2 above)
Comment (clinique) [inline addons/binding/org.openhab.binding.netatmo/build.properties]: v 1.0.8 has been replaced by v 1.0.9 but I'm not sure I fully get your question

**Comment by clinique** [comment]
@martinvw : yes, I'm pending confirmation from another thread

File: addons/binding/org.openhab.binding.netatmo/build.properties (hunk 2 above)
Comment (martinvw) [inline addons/binding/org.openhab.binding.netatmo/build.properties]: It states that the (binary) file netatmo-java-retrofit-1.0.9.jar was added but not that the other one was removed, so if merged we end up with two files as can be seen in your repo:

https://github.com/clinique/openhab2-addons/tree/Issue2379/addons/binding/org.openhab.binding.netatmo/lib

Comment (clinique) [inline addons/binding/org.openhab.binding.netatmo/build.properties]: Ok, now it should be fine

**Comment by martinvw** [comment]
Just for completeness I and most likely @clinique are still waiting for confirmation that this actually solves all problems.

**Comment by martinvw** [comment]
Travis is happy again :-)

**Comment by martinvw** [comment]
thanks to all!

**Pull request merged**